src/core/api/label.go

```diff
@@ -221,6 +220,19 @@ func (l *LabelAPI) List() {
 
 // Put updates the label
 func (l *LabelAPI) Put() {
+	var hasPermission bool
```

Comment:
Looks like the `Put` `Delete` `ListResources` have dup code?

If it's too much effort to refactor it now, please open new issue to address this after it's merged.

Reply:
`requireAccess` added to fix this